omero/developers/PythonBlitzGateway.rst

```diff
@@ -80,6 +80,40 @@ loading).
        survivin
     >>> conn.close()
 
+
+Unicode
+"""""""
+
+When unwrapping omero.model rstring attributes as shown above,
+the BlitzObjectWrapper subclasses will decode the utf8-encoded
+string and return the resulting unicode string.
+
+However, some methods added to BlitzObjectWrappers
+will return strings as follows::
+
```

Comment:
Do you have a test or way to check if there is no other methods?
I think having a test with all the methods will be useful if we adjust something we will immediately know the impact

Reply:
A test with ALL the methods of every object in the BlitzGateway? This will include all the methods of the underlying omero.model objects if we want to test whether any are returning strings. This is not really feasible.
Probably the best solution is to add unicode characters into all the various integration and robot tests so we know that these are being handled correctly.

Reply:
The point I was making here is how can I confirm that the list here is valid?

Reply:
You can confirm that the listed methods return strings by using the python shell as shown above. I can add more examples to cover the whole list if needed.
However, confirming that no other methods return strings is tricky. The only way is to check the source code, which is what I did to get that list.
We could go through all our BlitzGateway tests and ```assert type(s) == unicode``` but that would be a lot of work and I don't think our existing tests have complete coverage of all omero.model.object methods.

Reply:
I understood that this list was not an example but a complete list of files.
At least the ones we have tests for

Reply:
The ```$ bin/omero shell --login``` in the PR description is the example. The list in the PR is a complete list that I compiled from reading the BlitzGateway source code. I haven't checked the tests coverage for these methods.